.tool-versions

```diff
@@ -0,0 +1 @@
+starknet-foundry 0.15.0
```

Comment:
I guess this file is not necessary, since we try to construct one in actions, right?

Reply:
done!